tests/integration/helpers/project.rs

```diff
@@ -3,9 +3,12 @@ use std::io::Read;
 use std::path::{Path, PathBuf};
 use std::str;
 
+use tempfile::TempDir;
+
 #[derive(Debug)]
 pub struct Project {
     pub root: PathBuf,
+    pub(crate) _tmpdir: Option<TempDir>,
```

Comment:
pls name it just `tmpdir` without the `_`, since it needs to be kept until the test is done, so that the drop impl of `TempDir` would cleanup the directory.

Reply:
obsolete due to other changes!